lib/bolt/pal/yaml_plan/evaluator.rb

```diff
@@ -127,16 +129,22 @@ def generate_manifest(resources)
           MANIFEST
         end
 
-        def apply_manifest(scope, target, manifest)
+        def apply_manifest(scope, targets, manifest)
           ast = @evaluator.parse_string(manifest)
           apply_block = ast.body.body
           applicator = Puppet.lookup(:apply_executor)
-          applicator.apply([target], apply_block, scope)
+          applicator.apply([targets], apply_block, scope)
         end
 
         # This is the method that Puppet calls to evaluate the plan. The name
         # makes more sense for .pp plans.
         def evaluate_block_with_bindings(closure_scope, args_hash, plan)
+          unless plan.steps.any? { |step| step.body.key?('targets') }
```

Comment:
This will skip the warning if any step is using `targets`. I think we actually want to check if any step _is_ using `target` (since some plan could have both a step with `target` and a step with `targets` and we still want to warn).

Reply:
Derp. Definitely a typo from being used to typing targets.

Reply:
Derp. Definitely a case of the Mondays.